ctapipe/image/muon/tests/test_processor.py

```diff
@@ -0,0 +1,13 @@
+from ctapipe.image import ImageProcessor
+from ctapipe.io import EventSource
+
+
+def test_processor(dl1_muon_file):
+    from ctapipe.image.muon import MuonProcessor
+
+    with EventSource(dl1_muon_file, focal_length_choice="EQUIVALENT") as source:
+        image_processor = ImageProcessor(source.subarray)
+        muon_processor = MuonProcessor(source.subarray)
+        for event in source:
```

Comment:
In accordance to #2024 this should 1) assert something and 2) make sure this iteration runs at least once